composer/spec/dependabot/composer/update_checker_spec.rb

```diff
@@ -528,10 +528,13 @@
 
       it { is_expected.to be_nil }
 
-      it "logs an error" do
-        allow(Dependabot.logger).to receive(:error)
-        Dependabot.logger.error
-        expect(Dependabot.logger).to have_received(:error).once
+      context "logs an error" do
+        before { allow(Dependabot.logger).to receive(:error) }
+
+        it do
+          is_expected.to be_nil
+          expect(Dependabot.logger).to have_received(:error).at_least(:twice)
+        end
```

Comment:
I'm curious why you broke this out as a separate section? Why not blend this with the existing test in line 529? 

The spacing of this suggestion is off, so please don't take it verbatim, but this illustrates the general concept... 

```suggestion
      before { allow(Dependabot.logger).to receive(:error) }
      it { is_expected.to be_nil }
      expect(Dependabot.logger).to have_received(:error).once
```